trace-sdk/src/androidTest/java/io/bitrise/trace/data/collector/device/DeviceLocaleDataCollectorInstrumentedTest.java

```diff
@@ -1,26 +1,57 @@
 package io.bitrise.trace.data.collector.device;
 
+import android.content.Context;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+
 import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.Locale;
 
-import io.bitrise.trace.data.dto.Data;
 import io.bitrise.trace.data.collector.BaseDataCollectorInstrumentedTest;
+import io.bitrise.trace.data.dto.Data;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
 
 /**
  * Instrumented tests for {@link DeviceLocaleDataCollector}.
  */
 public class DeviceLocaleDataCollectorInstrumentedTest extends BaseDataCollectorInstrumentedTest {
 
+    final DeviceLocaleDataCollector collector = new DeviceLocaleDataCollector(context);
+
     /**
      * Verifies that when {@link DeviceLocaleDataCollector#collectData()} is called, the content of the returned
      * {@link Data} should not be {@code null}.
      */
     @Test
     public void collectData_contentShouldBeNotNull() {
-        final DeviceLocaleDataCollector deviceLocaleDataCollector = new DeviceLocaleDataCollector(context);
-        assertThat(deviceLocaleDataCollector.collectData().getContent(), is(notNullValue()));
+        assertThat(collector.collectData().getContent(), is(notNullValue()));
     }
+
+    @Test
+    public void collectData_mockData() {
```

Comment:
This test verifies that we are getting the locale data from resources/configuration. It would fail if we get the locale from a different source. Do we need such test?

Reply:
I think the purpose of this was to make sure it matched a given input e.g. if the locale was set to "en" the data that came out should also be "en" and we hadn't changed it at all. But I agree, it maybe isn't as useful as the others - so it could be removed.